frontend/src/hooks/file-processing/useScheduleConverter.ts

```diff
@@ -57,11 +57,11 @@ export const useScheduleConverter = (): [
     let parsedFileContent = convertBinaryToObjectArray(fileContent);
     setScheduleSheet(parsedFileContent);
     if (Object.keys(parsedFileContent).length !== 0) {
-      const logic = new ScheduleLogic(parsedFileContent as Array<Object>);
-      setSchedule(logic.getScheduleModel());
+      const parser = new ScheduleParser(parsedFileContent as Array<Object>);
+      setScheduleModel(parser.getScheduleModel());
```

Comment:
@pectom 
Dziś zrozumiałem, `getScheduleModel` - to bardzo trafna nazwa. Dzięki ;)